src/Microsoft.FeatureManagement/Targeting/ContextualTargetingFilter.cs

```diff
@@ -156,6 +156,13 @@ private bool IsTargeted(string contextId, double percentage)
 
             double contextPercentage = (contextMarker / (double)uint.MaxValue) * 100;
 
+            //
+            // Handle edge case of exact 100 bucket
```

Comment:
If percentage is 100, you can immediately return true at the beginning of the method.

Reply:
If I do the immediate return, it will be ripped out and turned into something like this as a part of the variants changes. I think this is cleaner with that future in mind.

Reply:
This PR has no notion of variants.

Reply:
The only thing I can imagine variants would play into is whether or not we should have this PR. If we do want it, even though variants are on the horizon, I don't expect the implementation should depend on that future change

Reply:
My concern is based off the idea that at any point in time, any user should be able to visit the repo, and the code should make sense. Having PRs that mix in the concerns of future changes can lead to confusing point-in-time source code.